content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,494 @@
+(`amd64` or `arm64`) and select and install the latest versions of the software packages.
+
+**CRI-O**
+
+Visit the [releases](https://github.com/cri-o/cri-o/releases) page (external link).
+
+Download the static binary bundle script:
+
+```shell
+curl https://raw.githubusercontent.com/cri-o/packaging/main/get > crio-install
+```
+
+Run the installer script:
+
+```shell
+sudo bash crio-install
+```
+
+Enable and start the `crio` service:
+
+```shell
+sudo systemctl daemon-reload
+sudo systemctl enable --now crio.service
+```
+
+Quick test:
+
+```shell
+sudo systemctl is-active crio.service
+```
+
+The output is similar to:
+
+```
+active
+```
+
+Detailed service check:
+
+```shell
+sudo journalctl -f -u crio.service
+```
+
+Note: The `cri-o` installer also provides the `crictl` command to manage and inspect
+Pods, containers, and container images. The `crictl` command is out of the scope
+of this tutorial.
+
+### Install  network plugins
+
+The `cri-o` installer installs and configures the `cni-plugins` package. You can
+verify the installation running the following command:
+
+
+```shell
+/opt/cni/bin/bridge --version
+```
+
+The output is similar to:
+
+```
+CNI bridge plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+To check the default configuration:
+
+```shell
+cat /etc/cni/net.d/11-crio-ipv4-bridge.conflist
+```
+
+The output is similar to:
+
+```json
+{
+  "cniVersion": "1.0.0",
+  "name": "crio",
+  "plugins": [
+    {
+      "type": "bridge",
+      "bridge": "cni0",
+      "isGateway": true,
+      "ipMasq": true,
+      "hairpinMode": true,
+      "ipam": {
+        "type": "host-local",
+        "routes": [
+            { "dst": "0.0.0.0/0" }
+        ],
+        "ranges": [
+            [{ "subnet": "10.85.0.0/16" }]
+        ]
+      }
+    }
+  ]
+}
+```
+
+{{< note >}}
+Make sure that the default `subnet` range (`10.85.0.0/16`) does not overlap with
+one of your active networks. If there is an overlap, you can edit the file and change it
+accordingly. Restart the service after the change.
+{{< /note >}}
+
+### Download and set up the kubelet
+
+Download the [latest stable release](/releases/download/) of the Kubelet.
+
+{{< tabs name="download_kubelet" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubelet"
+{{< /tab >}}
+{{< /tabs >}}
+
+Configure:
+
+```shell
+sudo mkdir -p /etc/kubernetes/manifests
+```
+
+```shell
+sudo tee /etc/kubernetes/kubelet.yaml <<EOF
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+authentication:
+  webhook:
+    enabled: false # Do NOT use in production clusters!
+authorization:
+  mode: AlwaysAllow # Do NOT use in production clusters!
```

Comment:
should we say what needs to be configures in production if we say not to use this in production?

Reply:
Thank you for reviewing, @SergeyKanzhelev. 

In this tutorial, we don't cover the setup of a production-ready Kubelet. At the beginning of the tutorial, we specify that:

_This tutorial does not cover the details you need for running a resilient control plane._

and:

_The Kubelet configuration used for this tutorial is insecure by design and should not be used in a production environment._

So we don't include instructions on how to configure a secure Kubelet and leave those details to existing Kubernetes cluster setup documentation.

What do you think @sftim? Should we add content to cover what is needed to be configured for a production environment?

Reply:
Can we at least point to some documentation where to look next? I think it is a bad practice to say that something is not for production without any pointers where to look next.

Reply:
We could point people to https://kubernetes.io/docs/setup/production-environment/ - in theory.

Within that section we could one day document cluster setup using standalone kubelets, for example as part as a scalable control plane. I doubt we'll get that documented within 36 months from today, and I don't want to block this PR on the other documentation that might one day exist.

Reply:
Do we have any pointers on what those settings are used for? Maybe just point to the reference of kubelet config then